packages/firebase_ui_storage/doc/paginated-loading-controller.md

```diff
@@ -0,0 +1,42 @@
+# PaginatedLoadingController
+
+When loading large amounts of objects from the Firebase Storage, it makes sense to load them in batches, but requires handling of the pagination logic.
+
+Firebase UI Storage provides [`StorageListView`] and [`StorageGridView`] widgets that implement pagination logic for you, and allow to load objects from the Firebase Storage in batches while scrolling.
+
+In cases where you want to build a completely custom scroll view, but still have pagination logic handled – you can use [`PaginatedLoadingController`].
+
+```dart
```

Comment:
Could we add examples of using `.load()` and `.shouldLoatNextPage()` as well?

Reply:
88b5cd26f4965ba9dce6900c5598ebfb20d610ca